README.md

```diff
@@ -327,9 +327,49 @@ interface IProviderCallback {
 
 ## Optional Flags
 
-You can enable the following optional flags:
+### Disable Injected Provider
 
-- disableInjectedProvider: disable displaying injected provider as an option (default: false)
+By default is set to `false` and Web3Connect always displays InjectedProvider as an option to the user if available. However you can disable it as an optional flag if you desire:
+
+#### React Button
+
+```javascript
+<Web3Connect.Button disableInjectedProvider />
+```
+
+#### Core Module
+
+```javascript
+const web3Connect = new Web3Connect.Core({ disableInjectedProvider: true });
+```
+
+### Cache Provider
+
+By default is set to `false` and Web3Connect will always require the user to choose a provider option before triggering the onConnect event. However you can enable caching the last chosen provider. Example:
+
+#### React Button
+
+```javascript
+<Web3Connect.Button cacheProvider />
+```
+
+#### Core Module
+
+```javascript
+const web3Connect = new Web3Connect.Core({ cacheProvider: true });
+```
+
+There are four options for this flag: `false`, `true`, `onclick` and `onload`. Here are the expected behaviors for each one of them:
+
+- If `false`, it will always display the modal to choose a provider.
+- If `true` or `onclick`, it will connect to the last chosen provider on `toggleModal` or button click
+- If `onload`, it will connect to the last chosen provider on page load where Web3Connect is instantiated
+
+If you wish to reset the cached provider you can call the following method:
+
+```typescript
+web3Connect.clearCachedProvider();
+```
```

Comment:
Looks good, I will also be utilising the `reset` feature as well 👍 

My thoughts in general are that the cache should be enabled by default, explicitly disabling it if you want to force selection each time. Reasons being that it is assumed better UX if the user not always prompted and it remembers your selection between new tabs.

Reply:
The only reason I didn’t want to enable it by default is to not break backwards-compatibility